dags/pegs-challenge-submission-dag.py

```diff
@@ -0,0 +1,92 @@
+from datetime import datetime
+
+from airflow.decorators import dag, task
+from airflow.models import Param
+
+from orca.services.nextflowtower import NextflowTowerHook
+from orca.services.nextflowtower.models import LaunchInfo
+from orca.services.synapse import SynapseHook
+
+
+dag_params = {
+    "synapse_conn_id": Param("SYNAPSE_ORCA_SERVICE_ACCOUNT_CONN", type="string"),
+    "synapse_evaluation_id": Param("9615531", type="string"),
+    "tower_conn_id": Param("PEGS_CHALLENGE_PROJECT_TOWER_CONN", type="string"),
+    "tower_run_name": Param("pegs_model_submission_evaluation", type="string"),
+    "tower_view_id": Param("syn53239158", type="string"),
+    "tower_input_id": Param("syn53239289", type="string"),
+    "tower_compute_env_type": Param("spot", type="string"),
+}
+
+dag_config = {
+    "schedule_interval": None,
```

Comment:
Does this need to be automated to run on a schedule?

Reply:
Yes the plan is to run this DAG every minute. I've left this alone for now until the PEGS challenge official start date is announced. Then I will modify `start_date` and `schedule_interval`.